src/main/asciidoc/index.adoc

```diff
@@ -1,3 +1,125 @@
 = Cassandra Client for Vert.x
 
 A Vert.x client allowing applications to interact with a Cassandra service
+
+
+== Getting Started
+
+For now, in order to use the client you need to build it locally first:
+
+----
+git clone https://github.com/Sammers21/vertx-cassandra-client
+cd vertx-cassandra-client
+mvn clean install -Dmaven.test.skip=true -s .travis.maven.settings.xml
```

Comment:
Out of curiosity: why forcing a settings file? For snapshot repo definition?

Reply:
Will users be able to build without it?

Reply:
I'd like them to use the settings because it guaranteed that it will work fine

Reply:
Ok, fine